app/components/provider_interface/activity_log_event_component.rb

```diff
@@ -138,5 +118,41 @@ def interview_event_description(user, candidate)
 
       "#{user} updated interview with #{candidate}"
     end
+
+  private
+
+    def routes
+      @_routes ||= Rails.application.routes.url_helpers
+    end
+
+    def application_link
+      {
+        url: routes.provider_interface_application_choice_path(application_choice),
+        text: 'View application',
+      }
+    end
+
+    def offer_link
+      {
+        url: routes.provider_interface_application_choice_offer_path(application_choice),
+        text: 'View offer',
+      }
+    end
+
+    def feedback_link
+      {
+        url: routes.provider_interface_application_choice_feedback_path(application_choice),
+        text: 'View feedback',
+      }
+    end
+
+    def interview_link(audit)
+      return if audit.auditable.cancelled?
```

Comment:
Since this can now return `nil`, I think we need to update the conditional in the view to
`<% if link.present? %>`
https://github.com/DFE-Digital/apply-for-teacher-training/blob/master/app/components/provider_interface/activity_log_event_component.html.erb#L25

Reply:
That's a good point and one I thought I had tested!